src/main/java/org/opentripplanner/graph_builder/issue/service/DefaultDataImportIssueStore.java

```diff
@@ -26,12 +30,31 @@ public void add(DataImportIssue issue) {
 
   @Override
   public void add(String type, String message) {
-    add(Issue.issue(type, message));
+    if (fileName != null) {
+      add(Issue.issue(type, message + " - %s", fileName));
+    } else {
+      add(Issue.issue(type, message));
+    }
   }
 
   @Override
   public void add(String type, String message, Object... arguments) {
-    add(Issue.issue(type, message, arguments));
+    if (fileName != null) {
+      add(
+        Issue.issue(
+          type,
+          message + " - %s",
+          Stream.concat(Stream.of(arguments), Stream.of(fileName)).toArray()
+        )
+      );
+    } else {
+      add(Issue.issue(type, message, arguments));
+    }
+  }
+
+  @Override
+  public void setFilename(String filename) {
+    this.fileName = filename;
   }
```

Comment:
This is not working, we have almost 100 typed issues that will bypass this. I suggest you create a `IssueWithSource` witch take the source and an issue and decorate the messages by overriding the methods.

The in the `void add(DataImportIssue issue)` you can do if(...) { issue = new IssueWithSource(issue, currentSource); }